src/README.md

```diff
@@ -210,6 +210,70 @@ int main(int argc, char* argv[]) {
 }
 ```
 
+This Python example demonstrates custom detokenization with buferisation. The streamer receives 
+integer tokens corresponding to each word or subword, one by one. If tokens are decoded individually, 
+subwords will not be concatenated correctly, and the resulting text will lack appropriate spaces. 
```

Comment:
I believe the problem is not how strings are concatenated. The problems is that detokenize(tokenize(" a)) == a and tokenizer needs a larger context to avoid omitting spaces.

Reply:
by concatenation problems i meant exactly what you are saying. Because of `detokenize(tokenize(" a)) == a` words are concatenated without spaces.

Corrected to express this explicitly.